migration/migrator/migrations/course/20210130180157_git_main_repo.py

```diff
@@ -0,0 +1,31 @@
+import json
+from pathlib import Path
+
+def up(config, database, semester, course):
+
+    course_dir = Path(config.submitty['submitty_data_dir'], 'courses', semester, course)
+    config_file = Path(course_dir, 'config', 'config.json')
+
+    if config_file.is_file():
+
+        f = open(config_file, 'r')
+        j = json.load(f)
+
+        # add a default for the parameter 'git_autograding_branch',
+        # which indicates which branch should be used when grading
+        # student submissions by vcs/git
+        
+        # we'll default existing courses to 'master', which was the standard
+        # until mid 2020 -- new courses will default to 'main'
```

Comment:
This is confusing without quantifying it as the 2020-2021 academic year I think. Perhaps:

```suggestion
        # until spring 2021 -- new courses will default to 'main'
```
?

Reply:
I was referring to the git documentation it was August/September when the initial recommendation changed (its not finished yet).  It's not an academic year thing.

Reply:
ah. I would then ask to change the first sentence to end with `was the standard in git` just to be more clear on whose standard we're referring to.